contracts/Account.sol

```diff
@@ -573,6 +643,55 @@ contract Account is UUPSOwnableUpgradeable, UsingRegistryUpgradeable, Managed, I
         return scheduledVotes[group].toWithdrawFor[beneficiary];
     }
 
+    /**
+     * @notice Revokes votes from a validator group. It first attempts to revoke pending votes,
+     * and then active votes if necessary.
+     * @dev Reverts if `revokeAmount` exceeds the total number of pending and active votes for
+     * the group from this contract.
+     * @param group The group to withdraw CELO from.
+     * @param lesserAfterPendingRevoke Used by Election's `revokePending`. This is the group that
+     * is before `group` within the validators sorted LinkedList, or address(0) if there isn't one,
+     * after the revoke of pending votes has occurred.
+     * @param greaterAfterPendingRevoke Used by Election's `revokePending`. This is the group that
+     * is after `group` within the validators sorted LinkedList, or address(0) if there isn't one,
+     * after the revoke of pending votes has occurred.
+     * @param lesserAfterActiveRevoke Used by Election's `revokeActive`. This is the group that
+     * is before `group` within the validators sorted LinkedList, or address(0) if there isn't one,
+     * after the revoke of active votes has occurred.
+     * @param greaterAfterActiveRevoke Used by Election's `revokeActive`. This is the group that
+     * is after `group` within the validators sorted LinkedList, or address(0) if there isn't one,
+     * after the revoke of active votes has occurred.
+     * @param index Used by Election's `revokePending` and `revokeActive`. This is the index of
+     * `group` in the this contract's array of groups it is voting for.
+     */
+    function revokeTransferVotes(
```

Comment:
Is the "transfer" in the name useful? This is just a general function that revokes the votes we want to revoke - it doesn't care where those revocations had come from.
Potential naming suggestions: rename this to `revokeVotes` and the current `revokeVotes` to `_revokeVotes`, or rename this to `onlyRevokeVotes`.

Reply:
Updated to revokeAndScheduleVotes

Reply:
Updated to revokeAndScheduleVotes